client/src/state.js

```diff
@@ -576,7 +583,7 @@ export const store = _createStore({
       // })
       const resultsTemplate = _.template(SpaqlQuery, esTemplateOptions);
       //var sparql = self.state.queryTemplates[template_name]({'n': n, 'o': o, 'q': q})
-      var sparql = resultsTemplate({ n: n, o: o, q: q, rt: rt, exact: exact, minRelevance: minRelevance });
+      var sparql = resultsTemplate({ n: n, o: o, q: q, textMatchAll: match, rt: rt, exact: exact, minRelevance: minRelevance });
```

Comment:
oops, was there already and exact option, and just not implemented fully?

Reply:
not really. I introduced textMatchAll in state

Reply:
I know, but i think exact was in there for that purpose.
So, I think one of them can do away.

Reply:
I see what you meant. Updated the code to use searchExactMatch. I verified that it is not being used anywhere else